packages/sdk/src/base/storage/base.ts

```diff
@@ -4,23 +4,27 @@ export type StorageOptions = {
 	prefix?: string;
 };
 
+const KEY_AUTH_TOKEN: string = 'auth_token';
+const KEY_AUTH_REFRESH_TOKEN: string = 'auth_refresh_token';
+const KEY_AUTH_EXPIRES: string = 'auth_token';
```

Comment:
Lets use a TypeScript Enum:

```suggestion
enum Keys {
  AuthToken = 'auth_token',
  RefreshToken = 'auth_refresh_token',
  Expires = 'auth_expires'
}
```

(also noticed a typo here. KEY_AUTH_EXPIRES is set to "auth_token")

Reply:
Thats actually better +